### PR TITLE
3424 styling for banner

### PR DESCRIPTION
**Pull request by eastandwestwind**
Closes https://github.com/ethyca/fides/issues/3424

### Code Changes

* [ ] Updates banner styling according to Figma mocks

### Steps to Confirm

* [ ] Nav to http://localhost:3000/fides-js-components-demo.html
* [ ] See banner matches mocks (note that the mocks have a small screen width, and on a standard laptop, this will appear stretched out):

Mocks:
Screenshot 2023-06-01 at 5 41 30 PM

This PR:
Screenshot 2023-06-01 at 5 37 19 PM

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`
* [ ] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### Description Of Changes

_Write some things here about the changes and any potential caveats_

### CODE REVIEW

File: clients/fides-js/src/lib/overlay.module.css [hunk 1]
```diff
@@ -5,7 +5,8 @@
   /* Colors */
   --fides-consent-overlay-primary-color: #8243f2;
   --fides-consent-overlay-background-color: #f7fafc;
-  --fides-consent-overlay-font-color: #4a5568;
+  --fides-consent-overlay-font-color-light: #4a5568;
```
Comment (allisonking) [inline clients/fides-js/src/lib/overlay.module.css:8]: nit: can we keep this as `--fides-consent-overlay-font-color` and then have the other one just be `--fides-consent-overlay-font-color-dark` or even `--fides-consent-overlay-link-color`? `-light` makes me think it'll be light, not a body color 🙂

**Review by allisonking — APPROVED** [review]
this looks good overall! https://github.com/ethyca/fides/pull/3429 will fix the font 

font size is going to be an issue, and one I ran into in that PR as well. I opted to use `em`s which makes things look a little big I think, at least in the banner. the modal looks ok though. actually I think the figma might have different font sizes for the modal and the banner which makes it difficult to use the font size var 🤔 

let's get yours merged in and I can resolve conflicts in my branch!

File: clients/fides-js/src/lib/overlay.module.css (hunk 1 above)
Comment (eastandwestwind) [reply clients/fides-js/src/lib/overlay.module.css:8]: yes good idea, thanks for the review!

File: clients/fides-js/src/lib/overlay.module.css [hunk 2]
```diff
@@ -26,10 +27,13 @@
   );
   /* Text */
   --fides-consent-overlay-title-font-color: var(
-    --fides-consent-overlay-font-color
+    --ides-consent-overlay-font-color
```
Comment (allisonking) [inline clients/fides-js/src/lib/overlay.module.css:30]: whoops! looks like there are a few other instances too

```suggestion
    --fides-consent-overlay-font-color
```

**Review by allisonking — APPROVED** [review]
🎉

**Pull request merged**